cmd/scaffold.go

```diff
@@ -150,8 +151,8 @@ func (fnb *FlowNodeBuilder) BaseFlags() {
 	fnb.flags.StringVar(&fnb.BaseConfig.AdminClientCAs, "admin-client-certs", defaultConfig.AdminClientCAs, "admin client certs (for mutual TLS)")
 
 	fnb.flags.DurationVar(&fnb.BaseConfig.DNSCacheTTL, "dns-cache-ttl", defaultConfig.DNSCacheTTL, "time-to-live for dns cache")
-	fnb.flags.StringVar(&fnb.BaseConfig.LibP2PStreamCompression, "stream-compression", p2p.NoCompression,
-		"networking stream compression mechanism")
+	fnb.BaseConfig.PreferredUnicastProtocols = fnb.flags.StringSlice("preferred-unicasts", nil,
```

Comment:
why not StringSliceVar? [example](https://github.com/onflow/flow-go/blob/master/cmd/access/node_builder/access_node_builder.go#L563)
Also, can we call this `preffered-unicast-protocol`.
Also, do you know if there is a enum type option to read these command line?

Reply:
fixed in https://github.com/onflow/flow-go/pull/1643/commits/f68f085c988ed7717d0149d62e40aacf835bd274, however, could not find enum support for command line.